conans/test/unittests/model/version/test_version_range.py

```diff
@@ -5,6 +5,7 @@
 from conans.model.version_range import VersionRange
 
 values = [
+    ['=1.0.0',  [[['=', '1.0.0']]],                   ["1.0.0"],          ["0.1"]],
```

Comment:
so `[=1.0.0]` is supported?

Reply:
Yes, I was a bit surprised too, seems more an unexpected result of the parsing, but I am concerned that removing now might break someone, so probably not worth it.